components/infobox/commons/infobox_league.lua

```diff
@@ -251,21 +248,44 @@ function League:createInfobox()
 	local builtInfobox = self.infobox:widgetInjector(self:createWidgetInjector()):build(widgets)
 
 	if self:shouldStore(args) then
-		self.infobox:categories('Tournaments')
-		if not String.isEmpty(args.team_number) then
-			self.infobox:categories('Team Tournaments')
-		end
-		if String.isNotEmpty(args.player_number) or String.isNotEmpty(args.individual) then
-			self.infobox:categories('Individual Tournaments')
-		end
-		self.infobox:categories(unpack(self:getWikiCategories(args)))
+		self.infobox:categories(unpack(self:getCategories(args)))
 		self:_setLpdbData(args, links)
 		self:_setSeoTags(args)
 	end
 
 	return tostring(builtInfobox) .. WarningBox.displayAll(League.warnings)
 end
 
+--- Allows for overriding this functionality
+function League:getCategories(args)
+	self.infobox:categories('Tournaments')
+	if not String.isEmpty(args.team_number) then
+		self.infobox:categories('Team Tournaments')
+	end
+	if String.isNotEmpty(args.player_number) or String.isNotEmpty(args.individual) then
+		self.infobox:categories('Individual Tournaments')
+	end
+
+	local tier = args.liquipediatier
+	local tierType = args.liquipediatiertype
+
+	local tierCategory, tierTypeCategory = Tier.toCategory(tier, tierType)
+	local isValidTierTuple = Tier.isValid(tier, tierType)
+	self.infobox:categories(tierCategory)
+	self.infobox:categories(tierTypeCategory)
+
+	if not isValidTierTuple and not tierCategory and String.isNotEmpty(tier) then
+		table.insert(self.warnings, String.interpolate(INVALID_TIER_WARNING, {tierString = tier, tierMode = 'Tier'}))
+		self.infobox:categories('Pages with invalid Tier')
+	end
+	if not isValidTierTuple and not tierTypeCategory and String.isNotEmpty(tierType) then
+		table.insert(self.warnings, String.interpolate(INVALID_TIER_WARNING, {tierString = tierType, tierMode = 'Tiertype'}))
+		self.infobox:categories('Pages with invalid Tiertype')
+	end
```

Comment:
Personally i'd favor not using `infobox:categories` here directly, if the return value is already thrown in there. Was there a discussion somewhere?

Reply:
for the team/indiv stuff: it already sets them fixed via commons
for the tier stuff: if a wiki wants the tier categories adjusted they can just overwrite that function or the `Tier.toCategory` function, haven't found any yet though

but i am thinking of making the tier categories a sep. function anyways